Scalar/CommandLine/CloneVerb.cs

```diff
@@ -309,6 +309,15 @@ private Result DoClone(string fullEnlistmentRootPathParameter, string normalized
                 cloneResult = this.TryRegisterRepo();
             }
 
+            if (cloneResult.Success)
+            {
+                this.Output.WriteLine("Complete!");
+            }
+            else
+            {
+                this.Output.WriteLine("Complete with errors.");
```

Comment:
Perhaps "Process finished with errors." ?

Reply:
Do we want to also change "Complete" to something else? I was just trying to be consistent in the "done/complete/finished" wording. Happy to go with whatever.

Reply:
Your wording is fine. Ignore me and merge.

Reply:
You're probably having a nice, quiet evening. I'll merge ;)

Reply:
Thanks! We can always change this at a later date if needs be.